common/src/prover_state/persistence.rs

```diff
@@ -106,7 +106,11 @@ pub fn verifier_to_disk(circuit: &VerifierData, circuit_config: &CircuitConfig)
 }
 
 fn write_bytes_to_file(bytes: &[u8], path: String) {
-    let file = OpenOptions::new().write(true).create(true).open(path);
+    let file = OpenOptions::new()
+        .write(true)
+        .create(true)
+        .truncate(true)
+        .open(path);
```

Comment:
This is unrelated to the dependencies update, but latest clippy is complaining about setting `create` without setting `truncate` as well.